main.tf

```diff
@@ -72,6 +73,11 @@ resource "google_container_cluster" "cluster" {
     master_ipv4_cidr_block = var.master_ipv4_cidr_block
   }
 
+  # Enable the PodSecurityPolicy admission controller for the cluster.
+  pod_security_policy_config {
+    enabled = true
```

Comment:
Can this be set with a variable. We can default it on but makes it easier for people to disable if they have already moved to other solutions like Gatekeeper.